dpx.yml

```diff
@@ -16,10 +19,33 @@ services:
       - "./opt-apigateway:/opt"
     networks:
       - "webnet"
+    depends_on:
+      - "sql-db"
     logging:
       driver: "fluentd"
       options:
         tag: dpx-rest
+  sql-db:
+    image: postgres:11
+    deploy:
+      replicas: 1
+    ports:
+      - target: 5432
+        published: 5433
+        protocol: tcp
+        mode: host
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=secret
+      - PGDATA=/var/lib/postgresql/data/pgdata
+    volumes:
+      - "./pgdata:/var/lib/postgresql/data/pgdata"
```

Comment:
Are you sure that pgdata directory exists? Makefile must be updated in order to create this directory (make start) and clean it up (make distclean). Also maybe a clearer name, like rest-db would be better.

Reply:
Can you take another look? I haven't used makefile, so not sure if the changes are correct.

Reply:
I have fixed the mkdir with -p option as you said. But I don't see any problem with distclean - it works fine. Can you check it again?